src/encoding.rs

```diff
@@ -275,9 +349,14 @@ pub fn check_wire_type(expected: WireType, actual: WireType) -> Result<(), Decod
 
 /// Helper function which abstracts reading a length delimiter prefix followed
 /// by decoding values until the length of bytes is exhausted.
-pub fn merge_loop<T, M, B>(value: &mut T, buf: &mut B, mut merge: M) -> Result<(), DecodeError>
+pub fn merge_loop<T, M, B>(
```

Comment:
@danburkert  Do you consider this and other similar functions to be part of the API? Technically they are visible to users, but they are doc(hidden) and are not very user friendly.

Reply:
No, anything `doc(hidden)` is not part of the public API, including this entire module, and the doc(hidden) methods on `Message`.